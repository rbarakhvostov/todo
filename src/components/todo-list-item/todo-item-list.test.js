import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import TodoListItem from './todo-list-item';

describe('<App />', () => {
  it ('TodoListItem renders without crashing', () => {
    const wrapper = mount(<TodoListItem />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
