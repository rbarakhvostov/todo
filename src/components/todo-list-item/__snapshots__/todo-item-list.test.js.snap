// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> TodoListItem renders without crashing 1`] = `
<TodoListItem>
  <li
    className="todo-list-item"
  >
    <span
      className="todo-list-item-label"
    />
    <div
      className="wrap-todo-list-item-button"
    >
      <button
        className="todo-list-item-button"
        onClick={[Function]}
        type="button"
      >
        <span
          className="fa fa-exclamation"
        />
      </button>
      <button
        className="todo-list-item-button"
        onClick={[Function]}
        type="button"
      >
        <span
          className="fa fa-trash-o"
        />
      </button>
    </div>
  </li>
</TodoListItem>
`;
